app/Repositories/ReportbackRepository.php

```diff
@@ -20,24 +20,63 @@ public function __construct(AWS $aws)
      *
      * @todo Handle errors better during creation.
      * @param  array $data
-     * @return \Gladiator\Models\Message
+     * @return \Rogue\Models\Reportback
```

Comment:
Since this method can return null, we should specify that in the `@return` docblock:

```php
/**
 * ...
 * @return \Rogue\Models\Reportback|null
 */
```